### PR TITLE
Add HTML skeleton and implement flexbox

### PR DESCRIPTION
**Pull request by AlexHamn**
The HTML skeleton for the headline section is complete.
Created a CSS file with full implementation of flexbox.
The styling for all text is correctly configured.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
-# portfolio
-My personal portfolio with my most relevant projects.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> My personal portfolio with my most relevant projects.
+
+Here you will eventually find some awesome projects built by yours truly.
+
```
Comment (bellom) [inline README.md:8]: Good job setting up this README file.
If check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

kindly add the following;
- setup

Example of a good setup that helps the non-technical user to set up your app easily below;
```
Example: 
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- open `index.html` with live server
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="container">
+      <p>Alex Hamn</p>
```
Comment (bellom) [inline index.html]: Following the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
 `Your name` should be wrap in an `h1` tag which is the main title of the page so it is good practice to use `h1` tag.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @AlexHamn  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

1. Kindly check the comments under the review.

2. Great simple design 👍  but we need to stick to Microverse curriculum plan. so that we can build the skills needed to land a job.
image

Original Design:
image


let's take one step back, kindly go to the [Project Requirement Page](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) under Project requirements.
_You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using the templates in this [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlexHamn** [comment]
Would you mind telling me how to interpret the linter errors? 🙏🏼 
Cause I don't have the first clue about what all the error messages are trying to say.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body class="container bod">
+    <header class="container navigation">
+      <h1 class="logo">Alex Hamn</h1>
+      <i class="material-icons">reorder</i>
+    </header>
+    <main>
```
Comment (ahmetbozaci) [inline index.html]: **Mandatory** 
- [ ] Like last TSE review you should follow original design. You can use background image for that.

**OPTIONAL**
- [ ] Also your navigation bar has line original page has not. You can fix this too.


**Yours:**
image
___
**Original:**
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 


## Hello @AlexHamn ✋🏼  Nice done with working on last review.However there are still some issues you should fix. Please fix them then your project will be approved.





### ♻️ Request changes ♻️
 
- [ ] You should follow the design.

_**Check the comment about that  under the review.**_








### OPTIONAL SUGGESTIONS

#### These are optional 

### Cheers and Happy coding! 💻 👏
cat on laptop
___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

File: index.html (hunk 3 above)
Comment (AlexHamn) [reply index.html]: Hi Ahmet, thank you for your help. That is very strange; the code for the background images is in the CSS file, and also, in my browser, I don't see any line at the top of the page; you can see it in the attached screenshot. Would you mind explaining to me why this happens or how could I fix it? 🙏🏼 

ksnip_20210709-172818

File: README.md (hunk 1 above)
Comment (o-rashad) [reply README.md:8]: Thank you @AlexHamn for implementing the required changes ^^!

File: index.html (hunk 3 above)
Comment (o-rashad) [reply index.html]: Good work @AlexHamn the design looks good to me. 💯

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body class="container bod">
+    <header class="container navigation">
+      <h1 class="logo">Alex Hamn</h1>
```
Comment (o-rashad) [inline index.html]: - [ ] [OPTIONAL] Would it be nice if we made the logo linked to your home page? 🤔 

Try to implement that using an `<a>` tag 👍🏼 

That would enhance your UX. 💹👥

**Review by o-rashad — APPROVED** [review]
Hi @AlexHamn , 👋🏼 ^^!

# Approved 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes

_Kindly check the inline comments 👍🏼_ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**